### PR TITLE
Notebooks for Diffuser Community Scripts-2

### PR DESCRIPTION
**Pull request by ParagEkbote**
# What does this PR do?

I've created 4 New Notebooks for Diffuser Community Scripts, same as #525.  After this PR gets merged, I will update the Colab column in the ReadMe.

Please let me know if any improvements are needed and I will make the necessary changes.

Reference: https://github.com/huggingface/diffusers/blob/main/examples/community/README.md

## Who can review?

@stevhliu

### CODE REVIEW

File: diffusers/insta_flow.ipynb [hunk 1]
```diff
@@ -0,0 +1,486 @@
+{
```
Comment (stevhliu) [inline diffusers/insta_flow.ipynb:1]: <p>Missing some text about combining it with LoRA.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/527/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

**Review by stevhliu — COMMENTED** [review]
Thanks for adding notebooks where there are currently none!

**Comment by ParagEkbote** [comment]
I have updated the notebooks as per given feedback.

Could you please review?

cc: @stevhliu

**Review by stevhliu — APPROVED** [review]
LGTM!

**Pull request merged**